flexget/plugins/list/ombi_list.py

```diff
@@ -231,17 +230,36 @@ def items(self):
                                             self._items.append(entry)
         return self._items
 
+    def add_movie(self, entry):
+        log.verbose('Adding entry %s', entry.get('title'))
```

Comment:
We prefer to use brackets for values we _know_ will be present. A valid entry will _always_ have a title and url, so this should be `entry['title']`.